src/cmds/cloud/createSessionToken.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import { Client } from '@cesarbr/knot-cloud-sdk-js';
+import options from './options';
+
+const createSessionToken = (args) => {
+  const client = new Client({
+    protocol: args.protocol,
+    hostname: args.server,
+    port: args.port,
+    pathName: args.pathName,
+    id: args['client-id'],
+    token: args['client-token'],
+  });
+
+  client.on('ready', () => {
+    client.createSessionToken(args.id);
+  });
+  client.on('created', (token) => {
+    console.log({
```

Comment:
why did you break the line?
console.log({ token });

Reply:
I don't know kkkkk. Done.